Bottom: 9b24967b175b9f3d0c0f4c6db69582e9d6fa2ea9
Top:    5b702dac7485a3a660ef8c2611b83386f626b571
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 01:13:25 +0100

Refresh of hid-provided-non-crash-default

---

diff --git a/src/hid/common/hidnogui.c b/src/hid/common/hidnogui.c
index cc7d2fc..c146fd2 100644
--- a/src/hid/common/hidnogui.c
+++ b/src/hid/common/hidnogui.c
@@ -112,7 +112,6 @@ nogui_set_draw_xor (hidGC gc, int xor_)
 static void
 nogui_set_draw_faded (hidGC gc, int faded)
 {
-  CRASH;
 }
 
 static void
@@ -510,6 +509,7 @@ apply_default_hid (HID * d, HID * s)
   AD (set_line_cap);
   AD (set_line_width);
   AD (set_draw_xor);
+  AD (set_draw_faded);
   AD (draw_line);
   AD (draw_arc);
   AD (fill_circle);
